### PR TITLE
[PR-12076] Ability to natively share via email features header

### PR DESCRIPTION
**Pull request by dgtv**
## Related Stories

[](https://talkable.atlassian.net/browse/PR-12076)

### CODE REVIEW

File: sdk/build.gradle [hunk 1]
```diff
@@ -10,8 +10,8 @@ android {
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 27
-        versionCode 38
-        versionName "0.5.9"
+        versionCode 39
+        versionName "0.5.10"
```
Comment (zhuravel) [inline sdk/build.gradle]: Should these versions be in sync?
https://github.com/talkable/android-sdk/blob/5808fcdfa1456236dac8982bc24418e26b9dac4a/app/build.gradle#L21-L22

Comment (dgtv) [reply sdk/build.gradle]: No, this is needed only when `app` code is changed

**Comment by dgtv** [comment]
@vitalikdanchenko up

**Comment by dgtv** [comment]
@vitalikdanchenko I've done some refactoring, could you take a look one more time please?

**Pull request merged**